src/cli_plugin/install/downloaders/http.js

```diff
@@ -2,11 +2,39 @@ import Wreck from 'wreck';
 import Progress from '../progress';
 import { fromNode as fn } from 'bluebird';
 import { createWriteStream } from 'fs';
+import HttpProxyAgent from 'http-proxy-agent';
+import URL from 'url';
+
+function getProxyAgent(sourceUrl) {
+  const httpProxy = process.env.HTTP_PROXY;
```

Comment:
Unfortunately the case of the env variable is not really specified. On Linux most programs use the lower case variant `http_proxy` (and same for `no_proxy`). Most programs support either lower or upper case and I think we should also try to use lower-case (preferred) and fallback to upper-case.